src/python3_pip_skeleton/__main__.py

```diff
@@ -160,11 +160,89 @@ def new(args):
     )
 
 
-cfg_issue = """Missing parameter in setup.cfg. Expected format:
+cfg_issue = """Missing parameter in setup.cfg/pyproject.toml/.git, Expected format:
+------- setup.cfg
 [metadata]
 name = example
 author = Firstname Lastname
-author_email = email@address.com"""
+author_email = email@address.com
+
+------- pyproject.toml
+[[project.authors]]
+name = "Firstname Lastname"
+email = "email@address.com"
+"""
+
+
+def obtain_author_name_email(path: Path) -> tuple:
+    author: str = ""
+    author_email: str = ""
+    file_path_setup_cfg: Path = path / "setup.cfg"
+    file_path_pyproject_toml: Path = path / "pyproject.toml"
+
+    # Parse for an author name, email. The order of preference used is
+    # setup.cfg -> pyproject.toml -> .git -> user input.
+    # Author and Email are recieved together to avoid mismatches from
+    # obtaining in different places.
+
+    conf = ConfigParser()
+
+    if file_path_setup_cfg.is_file():
+        try:
+            conf.read(file_path_setup_cfg)
+
+            if "metadata" in conf:
+                if "author" in conf["metadata"]:
+                    author = conf["metadata"]["author"]
+                if "author_email" in conf["metadata"]:
+                    author_email = conf["metadata"]["author_email"]
+        except Exception as exception:
+            print(
+                "\033[1mUnable to parse setup.cfg because of the following error, "
+                "will try other sources:\033[0m"
+            )
+            print(exception)
+            print()
+
+    if (not author or not author_email) and file_path_pyproject_toml.is_file():
+        try:
+            conf.read(file_path_pyproject_toml)
```

Comment:
Please could you use https://pypi.org/project/tomli/ rather than ConfigParser to load toml. (We could use https://docs.python.org/3.11/library/tomllib.html but we still need to support py3.8-3.10)